analysis/parser/RawTextParser/build.gradle

```diff
@@ -8,6 +8,7 @@ dependencies {
     implementation group: 'org.json', name: 'json', version: json_version
     implementation group: 'com.google.code.gson', name: 'gson', version: gson_version
     implementation group: 'com.github.kotlin-inquirer', name: 'kotlin-inquirer', version: kotlin_inquirer_version
+    implementation project(path: ':import:SourceCodeParser')
```

Comment:
This should not be needed.
Dependencies between parsers should be avoided if there is no good reason for it.